Readers/interface/setLeptonSF3.h

```diff
@@ -0,0 +1,35 @@
+#ifndef TallinnNtupleProducer_Readers_setLeptonSF3_h
+#define TallinnNtupleProducer_Readers_setLeptonSF3_h
+
+#include "TallinnNtupleProducer/Objects/interface/RecoJetAK8.h"
+
+#include <vector>                                                // std::vector
+
+template <typename T>
+void
+setLeptonSF3(std::vector<T> & particles,
+                const std::vector<RecoJetAK8> & jets)
+{
+  if ( particles.size() )
+  {
+    bool is_electron = std::fabs(particles.at(0).pdgId()) == 11;
```

Comment:
Add a comment that `particles` refers to collection of leptons that contains only electrons or only muons. Consider renaming `particles` to `leptons`.

Reply:
done